react_frontend/src/assets/SearchBar.css

```diff
@@ -0,0 +1,24 @@
+.search-bar{
```

Comment:
LGTM for our MVP but we may want to change the units to more responsive later on.

Reply:
yeah, I realized that as well. The search bar seems to be functional when the screen is at full screen size, but it is not for half or quarter screen sizes. I will try to fix this as an issue for a later sprint if we have time

Reply:
I think for the spreadsheet page we'll have to make sure we're using 👍🏾